src/BuffCommand.cpp

```diff
@@ -49,6 +49,12 @@ class KargatumCS_BuffCOmmand : public CommandScript
 	{
 		Player* player = handler->GetSession()->GetPlayer();
 		std::string ArgStr = (char*)args;
+        
+        if (sConfigMgr->GetIntDefault("BuffCommand.Enable", 1) == 0) {
+            handler->SendSysMessage("The command is currently disabled");
+            handler->SetSentErrorMessage(true);
```

Comment:
what does this do?

Reply:
If you talking about `SetSentErrorMessage`, because we return false as the command did not successfully run we will get some error messages in the chat along side our message from `SensSysMessage` like all the commands available for that command or that the command doesn't exist. We don't want that, we only want to show the message form `SendSysMessage`

Reply:
ok seems fair, ty